tests/evaluate/test_tbrom.py

```diff
@@ -792,11 +799,18 @@ def test_generate_snapshot_on_named_selection_with_tbrom_is_ok(self):
         # Generate snapshot on named selection
         ns = twinmodel.get_named_selections(romname)
         snp_vec_ns = twinmodel.generate_snapshot(romname, False, named_selection=ns[0])
-        assert len(snp_vec_ns) == 78594
+        assert (
+            len(
+                snp_vec_ns.reshape(
+                    -1,
+                )
+            )
+            == 78594
+        )
         if sys.platform != "linux":
             # TODO - Fix BUG881733
-            assert np.isclose(snp_vec_ns[0], 1.7188266859172047e-05)
-            assert np.isclose(snp_vec_ns[-1], -1.5316792773713332e-05)
+            assert np.isclose(snp_vec_ns[0, 0], 1.7188266859172047e-05)
```

Comment:
Don't the changes from line 812 and 813 solve issue encounter on linux (BUG881733)?

Reply:
the Twin/TBROM considered in that test case has a vector field as output (deformation). Therefore the original formulation did not work as snp_vec_ns[0] or [-1] returned an array of 3 elements

Reply:
yes, I got that. I'm just wondering if the issue that was reported in TFS BUG881733 was not due to the original formulation and if that change was not fixing it (typically are the unit test passing on Ubuntu if we remove the if statement of line 807?)